functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/MisspelledStrategiesFunctionalTest.java

```diff
@@ -0,0 +1,53 @@
+package org.arquillian.smart.testing.ftest.configuration;
+
+import org.arquillian.smart.testing.ftest.testbed.configuration.Strategy;
+import org.arquillian.smart.testing.ftest.testbed.project.Project;
+import org.arquillian.smart.testing.rules.TestBed;
+import org.arquillian.smart.testing.rules.git.GitClone;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.arquillian.smart.testing.Configuration.SMART_TESTING_DEBUG;
+import static org.arquillian.smart.testing.ftest.testbed.TestRepository.testRepository;
+import static org.arquillian.smart.testing.ftest.testbed.configuration.Mode.ORDERING;
+import static org.arquillian.smart.testing.scm.ScmRunnerProperties.COMMIT;
+import static org.arquillian.smart.testing.scm.ScmRunnerProperties.PREVIOUS_COMMIT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MisspelledStrategiesFunctionalTest {
```

Comment:
Do we need a functional test for this feature? I am thinking in terms of pipeline performance but also in terms of that maybe unit tests here is enough

Reply:
Depends how we want to evolve this feature. If we want to have autocorrect then this would be good to have. Also the build here takes few seconds really.

Reply:
ok